GOVERNANCE.rst

```diff
@@ -384,27 +414,41 @@ If the proposed change cannot gather a 2/3 majority of the votes cast,
 the CC tries to resolve the deadlock.
 
 -  The CC will use consensus seeking.
--  If no consensus can be found within a month, the decision is made
-   through a simple majority vote (with tie breaking) among the CC
+-  If no consensus can be found within twenty working days excluding weekends
+   since the beginning of the stage-1 "reasonable time for consideration" period,
+   the decision is made through a simple majority vote (with tie breaking) among the CC
    members.
--  Any CC decision must be supported by an :ref:`steps`, which has been made public and discussed before the vote.
+-  If this vote is needed, the vote should take place within twenty-five working days after the beginning
+   of the stage-1 "reasonable time for consideration" period, during a regular, minuted CC meeting.
```

Comment:
```suggestion
```

Reply:
why would you want to remove this very generous time limit? It would introduce a problematic loophole where decision making can stall simply since the cc never votes on the PR.

We want to avoid dynamics where indefinite stalling of a decision is a viable option.

Reply:
I know, but it just adds to many conditions/rules/timescales which are hard to keep track of, the CC will be working to resolve conflicts, don't think we need to impose more time limits here

Reply:
ok, fine. Let's discuss this addition separately then, I still think giving timelines is important.